Formula/neovim.rb

```diff
@@ -0,0 +1,117 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+    sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+    resource "luarocks" do
+      url "https://github.com/luarocks/luarocks/archive/5d8a16526573b36d5b22aa74866120c998466697.tar.gz"
+      sha256 "cae709111c5701235770047dfd7169f66b82ae1c7b9b79207f9df0afb722bfd9"
+    end
+  end
+
+  head do
+    url "https://github.com/neovim/neovim.git", :shallow => false
```

Comment:
It would be great to check if that's still a problem but `:shallow => false` is fine if it is.